tailwind.config.cjs

```diff
@@ -6,6 +6,12 @@ module.exports = {
       fontFamily: {
         poppins: ['Poppins'],
       },
+      colors: {
+        't1-line': '#D18725',
+        't2-line': '#DF51A9',
+        't3-line': '#61D03F',
+        't4-line': '#68E3DF',
+      },
```

Comment:
you might need to persist them otherwise they could be purged.
see https://tailwindcss.com/docs/content-configuration#safelisting-classes

Reply:
Ah, I see, thanks!